payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AfterpayClearpayHeaderElement.kt

```diff
@@ -46,7 +46,7 @@ data class AfterpayClearpayHeaderElement(
             // The no break space will keep the afterpay logo and (i) on the same line.
```

Comment:
I noticed that on the after screenshot, it looks like the afterpay logo is actually on a new line now, so I think this change may have had an undesired effect? Can you look into that?

Reply:
ah gotcha, my bad for focusing on this specific part. It's still surprising to me that the logo is on a new line now -- I would've expected this change to not actually change the text's appearance

Reply:
Oh my apologies that's because the recordings are from separate emulators

Reply:
ahh got it. Thanks!!